acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -136,7 +136,7 @@ public void start(final String processName) {
     params.add("--chain-id");
     params.add(chainId);
     if (useDynamicPortAllocation) {
-      params.add("--data-path");
+      params.add("--data-directory");
```

Comment:
This was right, it should be --data-path. We changed this awhile back to be consistent with Pantheon

Reply:
👍